examples/streaming_transcribe_from_local_file.js

```diff
@@ -27,8 +27,15 @@ client.on('connect', connectionMessage => {
     console.log(`Connected with job id: ${connectionMessage.id}`);
 })
 
+// Optional config to be provided.
+const sessionConfig = new revai.SessionConfig(
+    /* (optional) metadata */ "this is an example", 
+    /* (optional) custom_vocabulary_id */ null, 
+    /* (optional) filter_profanity */ true
+);
+
 // Begin streaming session
-var stream = client.start();
+var stream = client.start(/* sessionConfig */);
```

Comment:
```suggestion
var stream = client.start(sessionConfig);
```

Reply:
Use the session config object you created with default values (the only non default value is metadata, but this option does not effect transcription so I believe it to be an acceptable value.